src/helpers/date-helpers.ts

```diff
@@ -45,3 +45,21 @@ export function getMonthName(month: number) {
 	}
 	return capitalizeFirstLetter(format(new Date(new Date().getFullYear(), month, 1, 0, 0, 0, 0), "MMMM", { locale: locale }));
 }
+
+export function titleForDateRange(intervalType: "Day" | "Week" | "Month", intervalStart: Date, variant?: "short" | "long") {
+	var duration: Duration = intervalType == "Month" ? { months: 1 } : intervalType == "Day" ? { days: 1 } : { weeks: 1 };
+	var intervalEnd = add(intervalStart, duration);
+
+	if (intervalType == "Month" && intervalStart.getDate() == 1) {
+		return `${getMonthName(intervalStart.getMonth())} ${intervalStart.getFullYear()}`;
+	}
+	else if (intervalType == "Week" || intervalType == "Month") {
+		return `${format(intervalStart, "MM/dd/yyyy")} - ${format(sub(intervalEnd, { days: 1 }), "MM/dd/yyyy")}`;
+	}
+	else if (intervalType == "Day") {
+		if (variant === "long") {
+			return `${getDayOfWeek(intervalStart)}, ${getFullDateString(intervalStart)}`;
```

Comment:
I know it is not part of the PR here, but any thoughts on using `do` in `getFullDateString` instead of `d`, so that we get 1st, 2nd, 3rd, etc. instead of 1, 2, 3, etc.

Reply:
Makes sense to me